letsencrypt/cli.py

```diff
@@ -642,8 +644,45 @@ def _setup_logging(args):
     logger.info("Saving debug log to %s", log_file_name)
 
 
-def main2(cli_args, args, config, plugins):
-    """Continued main script execution."""
+def _handle_exception(exc_type, exc_value, trace, args):
+    logger.debug(
+        "Exiting abnormally:\n%s",
+        "".join(traceback.format_exception(exc_type, exc_value, trace)))
+
+    if issubclass(exc_type, errors.Error) and (not args or not args.debug):
+        sys.exit(exc_value)
```

Comment:
Sometimes you would like to debug what exactly happened, but you would need to mess with the source code and comment out some stuff in order to do so. Since we need logging to be set up in order to write full traceback to `--logs-dir`, it's not an option to use it. But you can attempt to write the full traceback to the current directory, mimicking core dump files.